src/App.tsx

```diff
@@ -113,7 +113,7 @@ const App: FC = () => {
           <ScrollToContentButton contentRef={mainContentRef} text={t('skip_to_main_content')} />
           <ToastContainer autoClose={3000} hideProgressBar />
           <Header />
-          <StyledContent ref={mainContentRef} role="main" id="content">
+          <StyledContent tabIndex={0} ref={mainContentRef} role="main" id="content">
```

Comment:
ved å legge på en tab-index på en div, kan man sette fokus på elementet

Reply:
Forøvrig: kan du oppnå det samme med tabindex=-1? Nå må man trykke Tab en ekstra gang for å komme forbi elementet

Reply:
du er god, assa